docs/NNNN-schema-registry.md

```diff
@@ -0,0 +1,195 @@
+Extension Name: NNNN-schema-registry
+Authors: P. Cornwell, D. Granville
+Minimum OCFL Version: 1.0
+OCFL Community Extensions Version: 1.0
+Extension Version: 0.1
```

Comment:
I believe it was decided that extensions were not going to be versioned and that if an extension needed to be updated it should be superseded by a new extension.

Reply:
I agree, this was https://github.com/OCFL/extensions/issues/4#issuecomment-624064813 - should change to remove "Extension Version" line

Reply:
Thanks for clarifying this, will do.